core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/toDataFrame.kt

```diff
@@ -34,7 +34,7 @@ public inline fun <reified T> Iterable<T>.toDataFrame(vararg props: KProperty<*>
         properties(roots = props, maxDepth = maxDepth)
     }
 
-@Deprecated(DF_READ_DEPRECATION_MESSAGE, ReplaceWith(DF_READ_REPLACE_MESSAGE), DeprecationLevel.ERROR)
+@Deprecated(DF_READ_DEPRECATION_MESSAGE, ReplaceWith("this.unfold(columns)"), DeprecationLevel.ERROR)
```

Comment:
Move to constants with deprecation messages please, replace previous if it's incorrect

Reply:
It's just this one that needs to be fixed. Other functions have vararg argument and need *. This one has single lambda, thus no need for *.